ui/component/viewers/imageViewer.jsx

```diff
@@ -1,17 +1,30 @@
 // @flow
 import React from 'react';
+import loadImage from 'blueimp-load-image';
 
 type Props = {
   source: string,
 };
 
+function loadPic(source) {
+  loadImage(
+    source,
+    function(img, data) {
+      if (img.type === 'error') {
+        console.error('Cannot load image');
+      } else {
+        document.getElementsByClassName('file-render__viewer')[0].appendChild(img);
```

Comment:
You should move `loadPic` inside of the `ImageViewer` component so that it can use `refs` and dynamically inserting the image, instead of using `appendChild`

I think it's fine to use this library though.